nbactions.xml

```diff
@@ -1,17 +1,32 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<actions>
-        <action>
-            <actionName>run</actionName>
-            <packagings>
-                <packaging>jar</packaging>
-            </packagings>
-            <goals>
-                <goal>process-classes</goal>
-                <goal>org.codehaus.mojo:exec-maven-plugin:1.5.0:exec</goal>
-            </goals>
-            <properties>
-                <exec.args>-classpath %classpath com.lilithsthrone.main.Main</exec.args>
-                <exec.executable>java</exec.executable>
-            </properties>
-        </action>
-    </actions>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think this file should be in the change list?

Reply:
This was added to the gitignore, not sure how it got here. I will delete it but I have no idea if the next commit will add it back.